crates/store/re_types/tests/types/fuzzy.rs

```diff
@@ -236,42 +232,25 @@ fn roundtrip() {
             fuzzy22_1.clone(),
         );
 
-        #[rustfmt::skip]
-        let expected_extensions: HashMap<_, _> = [
-            ("fuzz1001", vec!["rerun.testing.components.AffixFuzzer1", "rerun.testing.datatypes.AffixFuzzer1"]),
-            ("fuzz1002", vec!["rerun.testing.components.AffixFuzzer2", "rerun.testing.datatypes.AffixFuzzer1"]),
-            ("fuzz1003", vec!["rerun.testing.components.AffixFuzzer3", "rerun.testing.datatypes.AffixFuzzer1"]),
-            ("fuzz1004", vec!["rerun.testing.components.AffixFuzzer4", "rerun.testing.datatypes.AffixFuzzer1"]),
-            ("fuzz1005", vec!["rerun.testing.components.AffixFuzzer5", "rerun.testing.datatypes.AffixFuzzer1"]),
-            ("fuzz1006", vec!["rerun.testing.components.AffixFuzzer6", "rerun.testing.datatypes.AffixFuzzer1"]),
-        ]
-        .into();
-
         eprintln!("arch = {arch:#?}");
-        let serialized = arch.to_arrow2().unwrap();
+        let serialized = arch.to_arrow().unwrap();
         for (field, array) in &serialized {
             // NOTE: Keep those around please, very useful when debugging.
             // eprintln!("field = {field:#?}");
             // eprintln!("array = {array:#?}");
-            if field.name == "rerun.testing.components.AffixFuzzer21" {
+            if field.name() == "rerun.testing.components.AffixFuzzer21" {
```

Comment:
that's still true for arrow1? :thinking: (here and everywhere else)

Reply:
Nope, good catch - fixed